finders/gps.py

```diff
@@ -1,20 +1,22 @@
 from dataclasses import dataclass
-from math import sin, cos, acos, pi, radians
+from math import sin, cos, acos, radians
+
 
 @dataclass
 class GPSPosition:
     latitude: float
     longitude: float
 
-def gps_distance(Start : GPSPosition,End : GPSPosition):
+
+def gps_distance(start: GPSPosition, end: GPSPosition):
```

Comment:
add doc string, for function it is something like this: 
```suggestion
def gps_distance(start: GPSPosition, end: GPSPosition) -> int:
   """
   Calculates the distance between two GPS points
   :param start: lat and long of start point
   :param end: lat and log of end point
   :return: distance between two points
   """

```